components/slider/index.css

```
@@ -133,49 +156,48 @@ governing permissions and limitations under the License.
 .spectrum-Slider-track {
   &:first-of-type {
     &:before {
-      border-start-start-radius: var(--spectrum-slider-track-border-radius);
-      border-end-start-radius: var(--spectrum-slider-track-border-radius);
-      border-start-end-radius: 0;
-      border-end-end-radius: 0;
+      border-start-start-radius: var(--mod-slider-track-corner-radius, var(--spectrum-slider-track-corner-radius));
+      border-end-start-radius: var(--mod-slider-track-corner-radius, var(--spectrum-slider-track-corner-radius));
     }
   }
 
   &:last-of-type {
     &:before {
-      border-start-start-radius: 0;
-      border-end-start-radius: 0;
-      border-start-end-radius: var(--spectrum-slider-track-border-radius);
-      border-end-end-radius: var(--spectrum-slider-track-border-radius);
+      border-start-end-radius: var(--mod-slider-track-corner-radius, var(--spectrum-slider-track-corner-radius));
+      border-end-end-radius: var(--mod-slider-track-corner-radius, var(--spectrum-slider-track-corner-radius));
     }
   }
 }
 
-.spectrum-Slider-track ~ .spectrum-Slider-track {
+.spectrum-Slider-track~.spectrum-Slider-track {
```

```suggestion
.spectrum-Slider-track ~ .spectrum-Slider-track {
```

^ suggest adding that spacing back for readability